### DISCUSSION TITLE
Emojis in file/folder names

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#116] [Feature Requests]
[Adam]: I use Emojis at the start of file/folder names to make it easier to find what I'm looking for. I can add emojis to names in Obsidian and see them in Zettel, but if I try to name them in Zettel, it just puts an underscore where I type an emoji. The rest of the app seems to support it just fine.

**Comment by nathanl** [comment]
I don't think filenames support apostrophes right now either.

**Comment by damionx7** [comment]
In the following comment, I have explained the reasoning for filename characters restriction.

https://github.com/damionx7/Zettel-Notes-Documentation/issues/46#issuecomment-1910770526
